opm/simulators/wells/WellInterface.hpp

```diff
@@ -190,7 +190,7 @@ class WellInterface : public WellInterfaceIndices<GetPropType<TypeTag, Propertie
     computeBhpAtThpLimitProdWithAlq(const Simulator& ebos_simulator,
                                     const SummaryState& summary_state,
                                     const Scalar alq_value,
-                                    DeferredLogger& deferred_logger) const = 0;
+                                    DeferredLogger& deferred_logger, bool iterate_if_no_solution = true) const = 0;
```

Comment:
I would avoid to have default arguments for virtual class and subclass, maybe other colleagues can weigh in here. @bska @akva2